examples/todo/src/models/todo.model.ts

```diff
@@ -44,3 +44,7 @@ export class Todo extends Entity {
     super(data);
   }
 }
+
+export interface TodoRelations {}
```

Comment:
In this example, `Todo` does not have any relations defined, thus `TodoRelations` should be an empty interface :+1:

To reduce noise in `git diff` output when adding new relations in the future, I am proposing to add a line comment into the empty object block as follows.

```ts
export interface TodoRelations {
  // describe navigational properties here
}
```

This should be applied to all places where we have an empty Relations interface and to the CLI template too.

Thoughts?

Reply:
I agree 👍